scripts_ZA/compareShapesForTTbarEstim.py

```diff
@@ -0,0 +1,298 @@
+#! /bin/env python
+
+import sys, os, json
+import copy
+import datetime
+import subprocess
+import numpy as np
+import glob
+import ROOT
+from ROOT import TCanvas, TPad, TLine
+
+import argparse
+
+
+
+def getSingleTop(path):
+
+    Mjj = ROOT.TH1F("Mjj", "Mjj", 40, 10, 1000)
+    Mlljj = ROOT.TH1F("Mlljj", "Mlljj", 50, 100, 1500)
+    Mll = ROOT.TH1F("Mll", "Mll", 75, 12, 252)
+    jet1pt = ROOT.TH1F("jet1pt", "jet1pt", 50, 20, 500)
+    jet2pt = ROOT.TH1F("jet2pt", "jet2pt", 50, 20, 300)
+    jjDR = ROOT.TH1F("jjDR", "jjDR", 50, 0, 6)
+    lep1pt = ROOT.TH1F("lep1pt", "lep1pt", 50, 20, 400)
+    lep2pt = ROOT.TH1F("lep2pt", "lep2pt", 50, 10, 200)
+    jjpt = ROOT.TH1F("jjpt", "jjpt", 50, 0, 450)
+    list_histos = []
+    lumi = 35922
+    
+    # I guess the Single Top should come from the MuEl category, since we are
+    # subtracting it to data which is in the MuEl category
```

Comment:
correct :smile:

Reply:
Hahaha, that was for me only, I forgot to remove it :D